src/zapv2/__init__.py

```diff
@@ -110,25 +114,22 @@ def urlopen(self, *args, **kwargs):
            - `args`:  all non-keyword arguments.
            - `kwargs`: all other keyword arguments.
         """
-        kwargs['proxies'] = self.__proxies
-        return urllib.urlopen(*args, **kwargs).read()
+        return self.session.get(*args, proxies=self.__proxies, **kwargs).text
```

Comment:
I deliberately didnt use the main session as we dont want to add the api key to requests that arent to the api - this method is typically used to request 3rd party urls.